spec/unit/pops/validator/validator_spec.rb

```diff
@@ -113,6 +113,12 @@ def with_environment(environment, env_params = {})
     end
   end
 
+  it 'should not raise error when one modulepath is a substring of another' do
+    with_environment(environment, modulepath: ['path', 'pathplus']) do
+      expect(validate(parse('class aaa::ccc() {}', 'pathplus/aaa/manifests/ccc.pp'))).not_to have_issue(Puppet::Pops::Issues::ILLEGAL_DEFINITION_LOCATION)
+    end
+  end
+
```

Comment:
Since modulepath is user-configurable, can you add a test case for when it ends with a trailing slash?

Reply:
@joshcooper added and tests passing.